staging/kos/pkg/registry/network/subnet/strategy.go

```diff
@@ -137,6 +137,10 @@ func (ss *subnetStrategy) checkNSAndCIDRConflicts(candidate *subnet.Summary) (er
 	// Check whether there are Namespace and CIDR conflicts with other subnets.
 	for _, potentialRival := range potentialRivals {
 		pr, err := subnet.NewSummary(potentialRival)
+		// Make sure potentialRival is well formed. The code in this file makes
+		// it impossible to create a malformed subnet, but the check is done in
+		// case this version of strategy is rolled out after a more lenient one
+		// which let malformed subnets through.
```

Comment:
See https://github.com/MikeSpreitzer/kube-examples/pull/69#discussion_r302189214. Your final recommendation was to remove the check below the added comment because we're not in production and because it's far from obvious why the check might be desirable. I propose to simply put an explanatory comment. If you still want the check removed I'll do that.